src/scss/components/_listings.scss

```diff
@@ -6,28 +6,59 @@
  *
  */
 
-$listing-borderColor: $color-border;
+$listing-background         : $white;
+$listing-promoBackground    : $orange;
+$listing-borderColor        : $color-border;
+$listing-iconFill           : $grey--mid;
+$listing-labelBackground    : $grey--offWhite;
+$listing-offlineBorderColor : $grey--lighter;
+$listing-offlineBackground  : $grey--lightest;
+$listing-borderRadius       : 4px;
 
 .c-listing {
-    padding-top: 0;
-    padding-bottom: 0;
+    padding: 0 spacing(x2);
+    background: $listing-background;
+    border-radius: $listing-borderRadius;
+    border: 1px solid $listing-borderColor;
+
+    @include media('<mid') {
+        border: 0;
+        padding: 0;
+        // Negative margins used to bring the listing container outside of the the grid containers to 
```

Comment:
Would you use the grid classes to do this instead? (`g-span` may work for this)

Hard to tell without the markup structure, so if you've got an example of that might be able to help see if it's possible.